apps/api/tests/db/index.test.ts

```diff
@@ -17,200 +17,40 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { eq } from 'drizzle-orm';
 import assert from 'node:assert';
 import { after, before, describe, it } from 'node:test';
 
 import { PostgreSqlContainer, StartedPostgreSqlContainer } from '@testcontainers/postgresql';
 
-import { connectToDb, type PostgresDb } from '@/db/index.js';
-import { actions } from '@/db/schemas/actions.js';
-import { agreements } from '@/db/schemas/agreements.js';
-import { applicationContents } from '@/db/schemas/applicationContents.js';
-import { collaborators } from '@/db/schemas/collaborators.js';
-import { files } from '@/db/schemas/files.js';
-import { revisionRequests } from '@/db/schemas/revisionRequests.js';
+import { connectToDb, getDbInstance, type PostgresDb } from '@/db/index.js';
 
-import { initTestMigration, PG_DATABASE, PG_PASSWORD, PG_USER } from '../testUtils.js';
+import { PG_DATABASE, PG_PASSWORD, PG_USER } from '../testUtils.js';
 
 describe('Postgres Database', () => {
 	let db: PostgresDb;
 	let container: StartedPostgreSqlContainer;
 
-	const user_id = 'testUser@oicr.on.ca';
-
 	before(async () => {
 		container = await new PostgreSqlContainer()
 			.withUsername(PG_USER)
 			.withPassword(PG_PASSWORD)
 			.withDatabase(PG_DATABASE)
 			.start();
-
-		const connectionString = container.getConnectionUri();
-
-		db = connectToDb(connectionString);
-
-		await initTestMigration(db);
 	});
 
 	describe('Connection', () => {
 		it('should connect successfully', () => {
-			assert.notEqual(db, undefined);
-		});
-	});
-
-	describe('Actions', () => {
```

Comment:
These tests were used for db setup and are essentially just testing that Drizzle is working, they can be removed for now